docs/tutorials/pytorch2onnx.md

```diff
@@ -78,19 +82,34 @@ python tools/deployment/pytorch2onnx.py \
 
-|    Model    |                          Config                          | Dynamic Shape | batch inference | Note  |
-| :---------: | :------------------------------------------------------: | :-----------: | :-------------: | :---: |
-|    FCOS     | `configs/fcos/fcos_r50_caffe_fpn_gn-head_4x4_1x_coco.py` |       Y       |        Y        |       |
-|     SSD     |               `configs/ssd/ssd300_coco.py`               |       Y       |        Y        |       |
-|   YOLOv3    |    `configs/yolo/yolov3_d53_mstrain-608_273e_coco.py`    |       Y       |        Y        |       |
-|    FSAF     |          `configs/fsaf/fsaf_r50_fpn_1x_coco.py`          |       Y       |        Y        |       |
-|  RetinaNet  |     `configs/retinanet/retinanet_r50_fpn_1x_coco.py`     |       Y       |        Y        |       |
-| Faster-RCNN |   `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py`   |       Y       |        Y        |       |
+|    Model    |                          Config                          | Dynamic Shape | batch inference | Note |
+| :---------: | :------------------------------------------------------: | :-----------: | :-------------: | :--: |
+|    FCOS     | `configs/fcos/fcos_r50_caffe_fpn_gn-head_4x4_1x_coco.py` |       Y       |        Y        |      |
+|     SSD     |               `configs/ssd/ssd300_coco.py`               |       Y       |        Y        |      |
+|   YOLOv3    |    `configs/yolo/yolov3_d53_mstrain-608_273e_coco.py`    |       Y       |        Y        |      |
+|    FSAF     |          `configs/fsaf/fsaf_r50_fpn_1x_coco.py`          |       Y       |        Y        |      |
+|  RetinaNet  |     `configs/retinanet/retinanet_r50_fpn_1x_coco.py`     |       Y       |        Y        |      |
+| Faster-RCNN |   `configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py`   |       Y       |        Y        |      |
 
 Notes:
 
 - *All models above are tested with Pytorch==1.6.0 and onnxruntime==1.5.1*
 
+- The parameters of Non Max Suppression in the above onnx model:
+
+  - `nms_pre`: The number of boxes before NMS when pytorch forward inference. In all the above models, `nms_pre` is `1000`.
+  - `deploy_nums_pre`: The number of boxes before NMS when exporting onnx files in torch. This parameter is `0` in all of the above models.
+  - `max_pre_img`: The number of boxes to be kept after NMS. Among the above models, only the `max_pre_img` parameter of faster_rcnn is `1000`, and the other models are all `200`.
+  - `max_output_boxes_pre_class`: Maximum number of output  boxes per class of NMS. The `max_output_boxes_pre` is `200` in the SSD model and is `100` in other models.
+
+- If the deployed backend platform is TensorRT, please add environment variables before running the file:
+
+  ```bash
+  export ONNX_BACKEND=MMCVTensorRT
+  ```
+
+- If you want to use the `--dynamic-shape` parameter in the TensorRT backend to export onnx, please delete the `--simplify` parameter, and vice versa.
```

Comment:
The Option is `--dynamic-export` in mmdet.

Reply:
Add reason why simplify and dynamic shape features cannot be used at same time here.